apps/storybook/src/ExpandableBlock.stories.tsx

```diff
@@ -29,29 +28,38 @@ export default {
 
 export const Basic: Story<Partial<ExpandableBlockProps>> = (args) => {
   return (
-    <ExpandableBlock title='Basic Block' {...args}>
-      {args.children}
+    <ExpandableBlock {...args}>
+      <ExpandableBlock.Header>
+        <ExpandableBlock.LabelArea>
```

Comment:
this is a lot of code for a basic block. Maybe a convenience `label` prop in `ExpandableBlock.Header` will help (similar to `Tile.Name`)

```jsx
<ExpandableBlock>
	<ExpandableBlock.Header label='Blah' />
	<ExpandableBlock.Content>
	 ...
	</ExpandableBlock.Content>
</ExpandableBlock>
```

Reply:
Done. I used the other subcomponent to keep the class structure the same.